src/pretix/plugins/stripe/templates/pretixplugins/stripe/checkout_payment_form_sepadirectdebit.html

```diff
@@ -0,0 +1,71 @@
+{% load i18n %}
+{% load bootstrap3 %}
+
+<div class="form-horizontal stripe-container">
+    {% if is_moto %}
+        <h1>
+            <span class="label label-info pull-right flip" data-toggle="tooltip_html" title="{% trans "This transaction will be marked as Mail Order/Telephone Order, exempting it from Strong Customer Authentication (SCA) whenever possible" %}">MOTO</span>
+        </h1>
+        <div class="clearfix"></div>
+    {% endif %}
+
+    <div class="stripe-errors sr-only">
+
+    </div>
+    <noscript>
+        <div class="alert alert-warning">
+            {% trans "For a SEPA Debit payment, please turn on JavaScript." %}
+        </div>
+    </noscript>
+
+    {% if request.session.payment_stripe_payment_method_id %}
+        <div id="stripe-current-card">
+            <p>{% blocktrans trimmed %}
+                You already entered a card number that we will use to charge the payment amount.
```

Comment:
This sounds wrong in a file called sepadirectdebit.html

Reply:
That was already fixed but not comitted... Pushing again...